### DISCUSSION TITLE
Import data, datetime, SQL Server, month data wrongly out of range

### DISCUSSION THREAD

**Discussion by bozo16** [dbeaver/dbeaver#8649] [Q&A] [answered]
DBeaver 6.3.0
Importing from csv into MS SQL Azure 12.0.2000.8, column of type DateTime
I have these dates in my file:
03/02/2020 *
13/01/2020
01/11/2019 *
27/09/2019
27/09/2019
20/09/2019
20/09/2019
* but only these dates are imported (committing every 1 row, skipping errors), and whether I specify Date/time format
dd/mm/yyyy[ HH:mm:ss[.SSS]] or
mm/dd/yyyy[ HH:mm:ss[.SSS]], they appear in the table as
2nd March 2020 and
11th Jan 2019
It should import them all as ddth Month yyyy.

**Comment by uslss** [comment]
hi @bozo16 ,
could you post csv sample and table ddl?

**Comment by bozo16** [comment]
-- DROP TABLE DBeaver_example

CREATE TABLE DBeaver_example
 ( Start_Date DATETIME NULL
 , End_Date   DATETIME NULL
 )
;

Start Date,End Date
,01/11/2019
,03/02/2020
,13/01/2020
,20/09/2019
,20/09/2019
,27/09/2019
,27/09/2019

**Comment by uslss** [comment]
thanks for the bug report

**Comment by serge-rider** [comment, marked as answer]
@bozo16 You cannot use `dd/mm/yyyy` or `mm/dd/yyyy` as date format. mm stands for minutes. MM stands for months.
